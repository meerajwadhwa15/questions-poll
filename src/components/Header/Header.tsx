import React, { FunctionComponent } from "react";

export type Props = {};

const Header: FunctionComponent<Props> = () => (
  <header>
    <h1>Question Poll</h1>
  </header>
);

export default Header;
