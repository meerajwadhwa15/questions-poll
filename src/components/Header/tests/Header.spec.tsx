import React from "react";
import Header from "../Header";
import { create } from "react-test-renderer";

const props = {
};

test("Matches the snapshot with Header", () => {
  const component = create(
      <Header {...props}>Header</Header>
  );
  expect(component.toJSON()).toMatchSnapshot();
});
