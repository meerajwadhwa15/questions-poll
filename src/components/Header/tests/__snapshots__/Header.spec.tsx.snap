// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot with Header 1`] = `
<header>
  <h1>
    Question Poll
  </h1>
</header>
`;
